Add reference table for Timeboost (#2074)
OffchainLabs/arbitrum-docs: arbitrum-docs/run-arbitrum-node/how-to-use-timeboost.mdx

> ### Pre-requisites: gather required information

Reviewer: ```suggestion
### Prerequisites: Gather the required information
```

import { AddressExplorerLink as AEL } from '@site/src/components/AddressExplorerLink';

Before we begin, make sure you have:

- Address of the auction contract
- Endpoint of the autonomous auctioneer

> The following table shows this information for Arbitrum Sepolia:

Reviewer 2: ```suggestion
The following table shows this information for the ArbitrumDAO-owned chains:
```

| Network          | Auction contract                                                              | Autonomous auctioneer endpoint  |
| ---------------- | ----------------------------------------------------------------------------- | ------------------------------- |

> | Arbitrum Sepolia | <AEL address="0x991DbEDf388CB5925318f06362D4fCa7b040527D" chainID={421614} /> | https://sepolia-bv.arbitrum.io/ |

Reviewer 2: ```suggestion
| Arbitrum Sepolia | <AEL address="0x991DbEDf388CB5925318f06362D4fCa7b040527D" chainID={421614} /> | https://sepolia-bv.arbitrum.io/ |
| Arbitrum One | Pending on-chain governance vote | Pending on-chain governance vote
| Arbitrum Nova | Pending on-chain governance vote | Pending on-chain governance vote
```

> ### Step 1: deposit funds into the auction contract

Reviewer: ```suggestion
### Step 1: Deposit funds into the auction contract
```
